packages/widget/src/components/Search/SearchInput.style.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, List, styled } from '@mui/material';
+import { Input as InputBase } from '../../components/Input.js';
+
+export const Input = styled(InputBase)(({ theme }) => ({
+  paddingRight: theme.spacing(1.5),
+}));
+
+interface SearchStickyContainerProps {
+  headerHeight: number;
+}
+
+export const searchContainerHeight = 64;
+
+// When the widget is in Full Height mode the
```

Comment:
Something is missing at the end of the comment?

Reply:
Have reword this